backend/src/main/java/se/chalmers/dat265/group1/api/order/OrderAPI.java

```diff
@@ -13,7 +13,7 @@
 
     Order getOrder(String orderID);
 
-    Order createNewOrder(Order order);
+    Order createNewOrder(Order order) throws InvalidDateFormatException;
 
-    Order updateOrder(String orderID, Order order);
+    Order updateOrder(String orderID, Order order) throws InvalidDateFormatException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'se.chalmers.dat265.group1.api.order.InvalidDateFormatException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)